### ISSUE TITLE
修改服务实例权重报500错误

### ISSUE THREAD

**Issue by 1424019919** [alibaba/nacos#10979]
![image](https://github.com/alibaba/nacos/assets/79009117/47dfb111-32e4-43e8-82be-bcd0d20afcfd)
在点击编辑修改权重时报下述错误
caused: errCode: 500,
errMsg: do metadata operation failed;caused: com.alibaba.nacos.consistency.exception.ConsistencyException: DEADLINE_EXCEEDED: deadline exceeded after 4.999978517 s.[
    remote_addr=pay-basic-msenacos-2.pay-basic-msenacos.paycenter.svc.pub-bjzdt.local/10.229 .129 .80: 7848

**Comment by 1424019919** [comment]
服务端版本是2.2.0，客户端版本是1.4.2

**Comment by xxzz1122** [comment]
Hope it would work 
https://zhuanlan.zhihu.com/p/634842906

**Comment by KomachiSion** [comment]
Your cluster node of nacos server can't connect each other. The error means your update request can't be sync to node of leader `pay-basic-msenacos-2.pay-basic-msenacos.paycenter.svc.pub-bjzdt.local` with timeout.

**Comment by KomachiSion** [comment]
You may check your deploy env for network and server status like CPU and GC.

**Comment by 1424019919** [comment]
> You may check your deploy env for network and server status like CPU and GC.
I have checked the CPU and memory, and they are fine. The status of the three deployed instances is also UP, and I was able to edit the instances normally before. However, this error suddenly occurred on Tuesday. I plan to restart the Nacos service, but I'm just curious about the reason for this error. I deployed it using the domain deployment method in Kubernetes, so there should be no issue with finding the corresponding IP. Therefore, I don't think it is necessary to delete the protocol file.
